[2024-11-12T22:52:53.159+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-12T22:52:53.189+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extractAPItoBronze.get_api manual__2024-11-12T21:23:43.667318+00:00 [queued]>
[2024-11-12T22:52:53.204+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extractAPItoBronze.get_api manual__2024-11-12T21:23:43.667318+00:00 [queued]>
[2024-11-12T22:52:53.205+0000] {taskinstance.py:2303} INFO - Starting attempt 27 of 27
[2024-11-12T22:52:53.228+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_api> on 2024-11-12 21:23:43.667318+00:00
[2024-11-12T22:52:53.237+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=936) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-12T22:52:53.241+0000] {standard_task_runner.py:63} INFO - Started process 938 to run task
[2024-11-12T22:52:53.242+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'extractAPItoBronze', 'get_api', 'manual__2024-11-12T21:23:43.667318+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/ExtracttoBronze.py', '--cfg-path', '/tmp/tmp0pgd4upk']
[2024-11-12T22:52:53.246+0000] {standard_task_runner.py:91} INFO - Job 153: Subtask get_api
[2024-11-12T22:52:53.316+0000] {task_command.py:426} INFO - Running <TaskInstance: extractAPItoBronze.get_api manual__2024-11-12T21:23:43.667318+00:00 [running]> on host bbd2fa8da241
[2024-11-12T22:52:53.448+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extractAPItoBronze' AIRFLOW_CTX_TASK_ID='get_api' AIRFLOW_CTX_EXECUTION_DATE='2024-11-12T21:23:43.667318+00:00' AIRFLOW_CTX_TRY_NUMBER='27' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-12T21:23:43.667318+00:00'
[2024-11-12T22:52:53.450+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-12T22:52:53.472+0000] {logging_mixin.py:188} INFO - https://api.openbrewerydb.org/breweries
[2024-11-12T22:53:16.653+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-12T22:53:16.654+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/ExtracttoBronze.py", line 40, in processing_api_json
    if df.empty:
       ^^
NameError: name 'df' is not defined
[2024-11-12T22:53:16.673+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=extractAPItoBronze, task_id=get_api, execution_date=20241112T212343, start_date=20241112T225253, end_date=20241112T225316
[2024-11-12T22:53:16.677+0000] {logging_mixin.py:188} INFO - b'\x9bJ\xb79\x1a \xfd\xf6\x066Xz\x8aW2\xfe\xc0\x8ds(\xd9%8\xc3\xe2\xa0\x83\xa7\x8e\xf9\xa6\x02'
[2024-11-12T22:53:16.682+0000] {logging_mixin.py:188} INFO - Senha final (sem padding): b'epcrcnckxddwcogu'
[2024-11-12T22:53:16.683+0000] {logging_mixin.py:188} INFO - epcrcnckxddwcogu
[2024-11-12T22:53:23.956+0000] {logging_mixin.py:188} INFO - Failed to send email: (535, b'5.7.139 Authentication unsuccessful, basic authentication is disabled. [CP2PR80CA0245.lamprd80.prod.outlook.com 2024-11-12T22:53:22.421Z 08DD02B8CDFE1EE6]')
[2024-11-12T22:53:23.977+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 153 for task get_api (name 'df' is not defined; 938)
[2024-11-12T22:53:24.033+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-11-12T22:53:24.076+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-12T22:53:24.083+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
